src/client/entity-editor/name-section/name-section.js

```diff
@@ -256,6 +223,60 @@ class NameSection extends React.Component {
 							onChange={onDisambiguationChange}
 						/>
 					</Col>
+					<Col>
```

Comment:
Same comment as above, we need the same column span definition here so that the Cols collapse on smaller screens